packages/backend/src/services/SavedChartsService/SavedChartService.ts

```diff
@@ -151,6 +171,10 @@ export class SavedChartService {
             properties: {
                 projectId: savedChart.projectUuid,
                 savedQueryId: savedChartUuid,
+                constantReferenceLinesNum: 0,
+                conditionalFormattingNumRules: 0,
+                showLegend: true,
+                margins: 'default',
```

Comment:
Doesn't make sense to have these properties in this event. 🤔 We only need it in the version creation.